bindings/cpp/session.cpp

```diff
@@ -2233,12 +2248,12 @@ error_info session::mix_states(const key &id, std::vector<int> &groups)
 }
 
 async_iterator_result session::start_iterator(const key &id, const std::vector<dnet_iterator_range>& ranges,
-								uint32_t type, uint64_t flags,
-								const dnet_time& time_begin, const dnet_time& time_end)
+		uint32_t type, uint64_t flags, const dnet_time& time_begin, const dnet_time& time_end)
```

Comment:
`start_iterator` shouldn't be used with new flags and type, so it is better to check it here and either fail it or unset new flags.